api/permissions_requests_handlers_test.go

```diff
@@ -0,0 +1,76 @@
+// +build integration
+
+package api_test
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strings"
+	"testing"
+
+	helpers "github.com/topfreegames/Will.IAM/testing"
+)
+
+func TestPermissionsRequestsListHandler(t *testing.T) {
+	helpers.CleanupPG(t)
+	saUC := helpers.GetServiceAccountsUseCase(t)
+	sa, err := saUC.CreateKeyPairType("some sa")
+	if err != nil {
+		t.Errorf("Unexpected error %s", err.Error())
```

Comment:
`t.Fatalf` ?

Reply:
great! changed in this file. Will change in others too (later)

Reply:
Please leave a TODO somewhere, specially if you plan on doing it in another PR.

Reply:
Done!